README.md

```diff
@@ -1,37 +1,98 @@
-# Unified Model Post-processing: um2nc-standalone
+# Unified Model to NetCDF Post-processing: um2nc
 
 ## About
 
-`um2nc-standalone` is an [ACCESS-NRI](https://www.access-nri.org.au/) project to merge multiple versions of unified model conversion tools to a single, canonical project for the ESM1.5 model. 
+`um2nc` is a `Python3` utility to convert [Unified Model data files](https://code.metoffice.gov.uk/doc/um/latest/papers/umdp_F03.pdf) to NetCDF format.
+
+The `um2nc-standalone` project is an [ACCESS-NRI](https://www.access-nri.org.au/) initiative to merge multiple versions of Unified Model NetCDF conversion tool to a single, canonical project. 
 
 ## Installation
 
-TODO
+### Gadi
 
-* `virtualenv` instructions
-* `conda`/`miniconda`/`micromamba?` instructions
+On Gadi, `um2nc` is available within the `vk83` `payu` environment. 
+To access it, run: 
+```
+module use /g/data/vk83/modules
+module load payu
+```
+> [!IMPORTANT]  
+> You need to be a member of the vk83 project on NCI to access the module. For more information check how to [Join relevant NCI projects](https://access-hive.org.au/getting_started/set_up_nci_account/#join-relevant-nci-projects)
 
-## User documentation
+### Local installation
+`um2nc` is available as a `conda` package in the [access-nri conda channel](https://anaconda.org/accessnri/um2nc).
+To install it run:
+```
+conda install accessnri::um2nc
+```
 
-TODO
+### Installing Dependencies
```

Comment:
Are these steps required? 

`udunits` can be installed as a conda package. Why should it be installed in other ways?
Also, is it not already installed with some of the dependencies (for example `iris`)?
@truth-quark @blimlim

Reply:
Great question. I have to admit my understanding of setting up conda packages and installations is very poor, so most of this goes over my head. I just tried following the install instructions on my mac omitting the `udunits` step (and made sure to uninstall my separate homebrew installation of `udunits`).

Everything worked out ok for me, and I'm able to run the tests and perform conversions. Conda installed `udunits2` probably via [`cf-units`](https://github.com/SciTools/cf-units/blob/main/requirements/cf-units.yml). 

@truth-quark do you know if this separate `udunits` installation is necessary?

Reply:
I think I managed to get it running earlier with the `conda` version, but I can confirm that.

Reply:
Doing `brew uninstall udunits` on my system doesn't break the tests, so I think the `conda` `udunits` has us covered.

Reply:
I will remove it for now.

Reply:
Removed in [4bc1add33f5e3890227cd5c38c17ac4be458b497](https://github.com/ACCESS-NRI/um2nc-standalone/pull/146/commits/4bc1add33f5e3890227cd5c38c17ac4be458b497)

Reply:
> @truth-quark do you know if this separate `udunits` installation is necessary?

I used it specifically for a `virtualenv` install, which `conda` seems to handle itself???

Reply:
Yes `conda` packages that need `udunits` (like `iris`) will install [udunits2](https://anaconda.org/conda-forge/udunits2) automatically.
We encourage people who want a local installation to use `conda` (`micromamba` even better).